### PR TITLE
Change 'container' to 'Pod' in Jobs documentation

### PR DESCRIPTION
**Pull request by weibeld**
This just mirrors the change of the following merged pull request: #30525

### CODE REVIEW

File: content/zh/docs/concepts/workloads/controllers/job.md [hunk 1]
```diff
@@ -441,7 +441,7 @@ other Pods for the Job failing around that time.
 当 Job 的 Pod 被删除时，或者 Pod 成功时没有其它 Pod 处于失败状态，失效回退的次数也会被重置（为 0）。
 
 <!--
-If your job has `restartPolicy = "OnFailure"`, keep in mind that your container running the Job
+If your job has `restartPolicy = "OnFailure"`, keep in mind that your Pod running the Job
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/job.md:444]: Line 450 "容器“ should be changed to "Pod"

**Comment by weibeld** [comment]
@tengqm I just made the change, let me know if this is correct.

**Pull request merged**